test/integration_test.py

```diff
@@ -324,8 +331,24 @@ def __check_manifest(self, file: IO[bytes], uuid_field_name: str) -> List[Mappin
         self.assertIn(uuid_field_name, reader.fieldnames)
         bundle_uuid = rows[0][uuid_field_name]
         self.assertEqual(bundle_uuid, str(uuid.UUID(bundle_uuid)))
+        # File is downloaded only once per test invocation since they can be quite large.
+        if not self.manifest_drs_uri_downloaded:
+            drs_field_name = first((f for f in reader.fieldnames if 'drs_uri' in f), None)
+            drs_uri = rows[0].get(drs_field_name)
+            if drs_uri:
+                self._test_tdr_download(drs_uri)
         return rows
 
+    def _test_tdr_download(self, drs_uri: str) -> None:
+        http = SAMClient().oauthed_http
```

Comment:
Hacky. I shouldn't have to instantiate a SAMClient only to steal one of its dependencies.